pkg/volume/local/local_test.go

```diff
@@ -76,6 +76,9 @@ func getPersistentPlugin(t *testing.T) (string, volume.PersistentVolumePlugin) {
 }
 
 func getTestVolume(readOnly bool, path string) *volume.Spec {
+	if err := os.MkdirAll(path, 0777); err != nil {
+		return nil
```

Comment:
Or what if you just use tmpDir directly instead of creating another subdir on top of it.